src/test/java/teammates/test/cases/action/InstructorFeedbackSubmissionEditSaveActionTest.java

```diff
@@ -35,15 +35,10 @@ protected String getActionUri() {
     }
 
     @Override
-    protected void prepareTestData() {
-        dataBundle = loadDataBundle("/InstructorFeedbackSubmissionEditSaveActionTest.json");
```

Comment:
The data loading code should remain in `prepareTestData`, as I think that's the way it was designed.